README.md

```diff
@@ -304,20 +304,17 @@ OpenShift LightSpeed (OLS) is an AI powered assistant that runs on OpenShift and
          In this case, file `postgres_password.txt` contains password required to connect to PostgreSQL. Also CA certificate can be specified using `postgres_ca_cert.crt` to verify trusted TLS connection with the server. All these files needs to be accessible. 
 
 ## 11. (Optional) Incorporating additional CA(s). You have the option to include an extra TLS certificate into the OLS trust store as follows.
-
-      ```yaml
+```yaml
       ols_config:
          extra_ca:
             - "path/to/cert_1.crt"
             - "path/to/cert_2.crt"
-      ```
+```
 
-      > This action may be required for self-hosted LLMs.
+ > This action may be required for self-hosted LLMs.
 
 ## 12. Registering new LLM provider
-    Please look info this
-    [contributing guide chapter](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel)
-    for more info.
+   Please look info this [contributing guide chapter](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
```

Comment:
Maybe change it for:
Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.

Reply:
@onmete updated accordingly